docs/en/engines/table-engines/special/merge.md

```diff
@@ -9,45 +9,54 @@ The `Merge` engine (not to be confused with `MergeTree`) does not store data its
 
 Reading is automatically parallelized. Writing to a table is not supported. When reading, the indexes of tables that are actually being read are used, if they exist.
 
-The `Merge` engine accepts parameters: the database name and a regular expression for tables.
-
-## Examples {#examples}
-
-Example 1:
+## Creating a Table {#creating-a-table}
 
 ``` sql
-Merge(hits, '^WatchLog')
+    CREATE TABLE ... Engine=Merge(db_name, tables_regexp)
 ```
 
-Data will be read from the tables in the `hits` database that have names that match the regular expression ‘`^WatchLog`’.
+**Engine Parameters**
+
+- `db_name` — Database name or a regular expression to match the DB names. You can use a constant expression that returns a string, for example, `currentDatabase()`. 
```

Comment:
May be worth mentioning here (not only in sql example), that database name as regular expression works only if adding `REGEXP(db_name)`, otherwise it is not treated as regexp.